packages/plugin/src/theia-proposed.d.ts

```diff
@@ -433,12 +433,6 @@ export module '@theia/plugin' {
          */
         useGlobalIgnoreFiles?: boolean;
 
-        /**
-         * Whether symlinks should be followed while searching.
-         * See the vscode setting `"search.followSymlinks"`.
-         */
-        followSymlinks?: boolean;
```

Comment:
These changes can be reverted, we can handle `followSymlinks` from the plugin-system in a separate pull-request.